src/mixins/cartMixin.js

```diff
@@ -67,6 +67,26 @@ export default {
                   ...DisplayableMoney
                 }
               }
+              discountCodes {
+                discountCode {
+                  id
+                  code
+                  name(locale: $locale)
+                  description(locale: $locale)
+                  cartDiscounts{
+                    value{
+                      ... on RelativeDiscountValue{
+                        permyriad
+                      }
+                      ... on AbsoluteDiscountValue{
+                        money{
+                          ...DisplayableMoney
+                        }
```

Comment:
Are you sure we need all these fields? Where else are you fetching/using the `description` or the `cartDiscounts`?